#!/bin/sh -e
##  by Joost Yervante Damad <joost@lumatec.be>

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

case "$1" in
       -patch) patch $patch_opts -p1 < $0;;
       -unpatch) patch $patch_opts -p1 -R < $0;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
                exit 1;;
esac

exit 0

@DPATCH@
--- openmsx-0.6.1.orig/build/detectsys.sh	2005-12-27 16:44:09.000000000 +0100
+++  openmsx-0.6.1/build/detectsys.sh	2006-07-31 21:49:25.000000000 +0200
@@ -37,6 +37,8 @@
 		OPENMSX_TARGET_CPU=x86_64;;
 	powerpc)
 		OPENMSX_TARGET_CPU=ppc;;
+	powerpc64)
+		OPENMSX_TARGET_CPU=ppc64;;
 	sparc*)
 		OPENMSX_TARGET_CPU=sparc;;
 	m68k)
